src/qililab/instruments/qblox/qblox_module.py

```diff
@@ -88,13 +80,14 @@ def __post_init__(self):
     settings: QbloxModuleSettings
     device: Pulsar | QcmQrm
     # Cache containing the last compiled pulse schedule for each sequencer
-    _cache: dict[int, PulseBusSchedule] = {}
+    cache: dict[int, PulseBusSchedule] = {}
 
     def __init__(self, settings: dict):
         # The sequences dictionary contains all the compiled sequences for each sequencer and a flag indicating whether
         # the sequence has been uploaded or not
-        self.sequences: dict[int, tuple[QpySequence, bool]] = {}  # {sequencer_idx: (program, True), ...}
+        self.sequences: dict[int, QpySequence] = {}  # {sequencer_idx: (program, True), ...}
         # TODO: Set this attribute during initialization of the instrument
+        # TODO: delete useless parameters nshots num bins since they are managed at compilation
```

Comment:
I could do it but then I would have to change more tests and base classes. Since having them there until we clean them doesn't crash anything I prefered to do that in a separate pr

Reply:
But I can do it here too otherwise